src/screens/OtherProfile/index.js

```diff
@@ -468,7 +393,7 @@ const OtherProfile = () => {
       blockUserRef.current.close();
       specificIssueRef.current.close();
       reportUserRef.current.close();
-      checkUserBlockHandle(dataMain.user_id);
+      refetchBlockStatus();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `checkUserBlockHandle` has been replaced with `refetchBlockStatus`. Ensure that `refetchBlockStatus` is providing the same functionality as `checkUserBlockHandle`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->